### PR TITLE
[Doc] Add doc for RotatE

### PR DESCRIPTION
**Pull request by classicsong**
*Issue #, if available:*
#980 

*Description of changes:*
Add doc for RotatE

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by zhjwy9343** [comment]
Can we have a preview link for these updated pages?

File: python/graphstorm/model/edge_decoder.py [hunk 1]
```diff
@@ -1351,8 +1351,8 @@ def out_dims(self):
         return 1
 
 class LinkPredictContrastiveRotatEDecoder(LinkPredictRotatEDecoder):
-    """ Decoder for link prediction designed for contrastive loss
-        using the RotatE as the score function.
+    f""" Decoder for link prediction designed for contrastive loss
+    using the RotatE as the score function.
```
Comment (zhjwy9343) [inline python/graphstorm/model/edge_decoder.py:1355]:
```suggestion
    f""" Decoder for link prediction designed for contrastive loss
            using the RotatE as the score function.
```

File: python/graphstorm/model/edge_decoder.py [hunk 2]
```diff
@@ -1351,8 +1351,8 @@ def out_dims(self):
         return 1
 
 class LinkPredictContrastiveRotatEDecoder(LinkPredictRotatEDecoder):
-    """ Decoder for link prediction designed for contrastive loss
-        using the RotatE as the score function.
+    f""" Decoder for link prediction designed for contrastive loss
```
Comment (zhjwy9343) [inline python/graphstorm/model/edge_decoder.py:1354]:
```suggestion
```

File: python/graphstorm/model/edge_decoder.py (hunk 1 above)
Comment (zhjwy9343) [reply python/graphstorm/model/edge_decoder.py:1355]: Aligning the 2nd line to the start of the first letter of the 1st line will resolve the different fonts issue.

File: docs/source/cli/model-training-inference/configuration-run.rst [hunk 3]
```diff
@@ -469,12 +471,12 @@ Link Prediction Task
...
     - Argument: ``--num-negative-edges-eval 1000``
     - Default value: ``1000``
-- **lp_decoder_type**: Set the decoder type for loss function in Link Prediction tasks. Currently GraphStorm support  ``dot_product`` and ``DistMult``.
+- **lp_decoder_type**: Set the decoder type for loss function in Link Prediction tasks. Currently GraphStorm support  ``dot_product``, ``distmult`` and ``rotate``.
 
     - Yaml: ``lp_decoder_type: dot_product``
     - Argument: ``--lp-decoder-type dot_product``
     - Default value: ``dot_product``
-- **gamma**: Gamma for ``DistMult``. The margin value in the score function.
+- **gamma**: Gamma for ``distmult`` and ``rotate``. A hyperparameter used to control the relation embedding initialization and the margin value in the score functions. See detials in :ref:`Computing Link Prediction Scores<link-prediction-score-func>`。
```
[the remark this answers is not in the document]
Comment (classicsong) [reply docs/source/cli/model-training-inference/configuration-run.rst:479]: The default in arguement.py is 12.
I have updated gammas to 12.

**Review by zhjwy9343 — APPROVED** [review]
LGTM

**Pull request merged**